test/command/suite/normalizers/nfkc121/unify_katakana_prolonged_sound_mark.test

```diff
@@ -0,0 +1,15 @@
+normalize \
+  'NormalizerNFKC121("unify_katakana_prolonged_sound_mark", true, \
+                     "report_source_offset", true)' \
+  "ァーアーィーイーゥーウーェーエーォーオー\
+ヵーカーガーキーギークーグーヶーケーゲーコーゴー\
+サーザーシージースーズーセーゼーソーゾー\
+ターダーチーヂーツーヅーテーデートードー\
+ナーニーヌーネーノー\
+ハーバーパーヒービーピーフーブープーヘーベーペーホーボーポー\
+マーミームーメーモー\
+ャーヤーューユーョーヨー\
+ラーリールーレーローヮー\
+ワーヰーヱーヲーンー\
+ヴーヷーヸーヹーヺー" \
```

Comment:
Could you split this to multiple test files?
This is difficult to debug.

We have small test files (one test file only tests one or a few features) instead of large test files. It's for easy to debug. If we have many data in one test file, it's difficult to set a break point and so on. For example, how to debug the `ポー` case?

Reply:
I have split to multiple test files.